composites/OnboardingWizard/onboarding-wizard.scss

```diff
@@ -121,8 +121,8 @@ body {
 			}
 
 			svg {
-				color: $color_purple;
-				fill: $color_purple;
+				color: $color_green;
+				fill: $color_green;
```

Comment:
Seem these properties don't do anything, a bit pointless even before the current changes. As far as I see, the step numbers color is passed in the configuration and applied as inline styles so the green won't be applied:

![screen shot 2017-09-20 at 09 53 08](https://user-images.githubusercontent.com/1682452/30632984-4d86015a-9dea-11e7-8587-43bbc35a3e92.png)

Re: `$color_green`, it has a very low contrast ratio with white: https://jdlsn.com/color/?type=hex&color=FFFFFF&color2=77b227. The result is 2.57:1 where it should be at least 4.5:1. Our greens are a bit tricky for a11y and we faced this issue also in other places. Maybe worth considering a global review of the greens. In this specific case, I'd suggest to use $color_green_medium     #008a00